orgs-using-d.dd

```diff
@@ -122,6 +122,13 @@ $(DIVC orgs-using-d center,
             $(FA_HIRING www.weka.io)
         )
     )
+    $(DORG Ecratum, http://www.ecratum.com, ecratum.png,
+        Supplier Relationship Management tool for SMBs,
```

Comment:
I had no idea what SMB means before looking it up. 

As you [can see](http://dtest.thecybershadow.net/artifact/website-49c3e5493bd6676bd644aea71cdc0ad75e35b292-dcbc8efc0159947d61ca38b8256bdd49/web/orgs-using-d.html), you already use two lines.
I did put some focus to have short and concise one-line description (it looks nicer and is more helpful to a reader). Maybe it's possible to shorten your description a bit and make it more informative?
Some ideas:

- Relationship management tooling
- Supplier management
- Supplier logistics
- Supplier relationships